pkg/cvc/v1alpha1/build.go

```diff
@@ -0,0 +1,282 @@
+/*
+Copyright 2019 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	apismaya "github.com/openebs/csi/pkg/apis/openebs.io/maya/v1alpha1"
+	errors "github.com/openebs/csi/pkg/generated/maya/errors/v1alpha1"
+	metav1 "k8s.io/api/core/v1"
+	resource "k8s.io/apimachinery/pkg/api/resource"
+)
+
+// Builder is the builder object for CStorVolume
+type Builder struct {
+	cvc  *CStorVolumeClaim
+	errs []error
+}
+
+// NewBuilder returns new instance of Builder
+func NewBuilder() *Builder {
+	return &Builder{
+		cvc: &CStorVolumeClaim{
+			object: &apismaya.CStorVolumeClaim{},
+		},
+	}
+}
+
+// BuilderFrom returns new instance of Builder
+// from the provided api instance
+func BuilderFrom(cvc *apismaya.CStorVolumeClaim) *Builder {
+	return &Builder{
+		cvc: &CStorVolumeClaim{
```

Comment:
Nil check for cvc

Reply:
If this is nil, then even if we have a check will the program not crash.

Reply:
updated with review comment